scanpy/tests/test_embedding.py

```diff
@@ -2,7 +2,7 @@
 
 import numpy as np
 import pytest
-from sklearn.utils.testing import assert_array_almost_equal
+from numpy.testing import assert_array_almost_equal
```

Comment:
scikit-learn imported this internally, and was throwing warnings that it will be removed from the scikit-learn public API.